.github/tests/lockdown/deps-values.yaml

```diff
@@ -0,0 +1,54 @@
+global:
```

Comment:
Where is this values file actually being used for a helm install? I don't get how this works. Looks like some changes are missing.

Reply:
Yeah. that line seems to have gotten dropped out during one of the merge conflicts. I've added it back.